app/services/catalog_cloner.rb

```diff
@@ -0,0 +1,28 @@
+class CatalogCloner
+  include ActiveModel::Model
+  include HasSlug
+
+  def self.validates_uniqueness_of(*args)
```

Comment:
je me dit que ce serait bien d'expliquer pourquoi cette méthode est vide et qu'on a faite le "exists" manuellement

Reply:
# this method is overrides HasSlug because validates_uniqueness_of is not present in Class Object